tools-containers/kdc-api-server/server/main.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"log"
+	"os"
+)
+
+func main() {
+
+	// Allow overriding of kadmin cli
+	kadminBin := os.Getenv("KADMIN_BIN")
+	if kadminBin == "" {
+		kadminBin = "kadmin"
+	}
+
+	// Allow overriding of ktutil cli
```

Comment:
from the code it seems that providing the path is mandatory (no default), but from the comment it seems like it's optional?

Reply:
It is optional. If missing this will be an empty string and handled differently later down the code path.

Reply:
What I mean is that if this is empty, then https://github.com/mesosphere/dcos-commons/pull/3049/files#diff-84c29347fc0e07a7e9f62417a4148736R137 will put an empty string into the `KAdminClient.KTUtilPath`, which is effectively a useless `KAdminClient` object, since in that case https://github.com/mesosphere/dcos-commons/pull/3049/files#diff-84c29347fc0e07a7e9f62417a4148736R554 will always return an error.

Why not handle this case much earlier, then?